config-model/src/test/java/com/yahoo/schema/processing/SummaryConsistencyTestCase.java

```diff
@@ -42,4 +43,26 @@ void attribute_combiner_transform_is_set_when_source_is_array_of_struct_with_onl
         Schema schema = ApplicationBuilder.createFromString(sd).getSchema();
         assertEquals(SummaryTransform.ATTRIBUTECOMBINER, schema.getSummaryField("elem_array_unfiltered").getTransform());
     }
+
+    @Test
+    @Disabled
+    void testDocumentTypesWithInheritanceOfNonExistingField() throws ParseException {
```

Comment:
Should this be `testDocumentSummaryWithInheritanceOfNonExistingSummary` ?

Reply:
Yes, thanks, will fix in next PR